crates/kubelet/src/server.rs

```diff
@@ -49,7 +62,21 @@ pub async fn start_webserver<T: 'static + Provider + Send + Sync>(
             }))
         }
     });
-    let server = Server::bind(address).serve(service);
+
+    let mut listener = TcpListener::bind(address).await.unwrap();
+    let mut incoming = listener.incoming();
+    let accept = hyper::server::accept::from_stream(stream! {
+        loop {
+            match incoming.next().await {
+                Some(Ok(stream)) => match acceptor.clone().accept(stream).await {
+                    result @ Ok(_) => yield result,
+                    Err(e) => break,
```

Comment:
Should we log and continue the loop instead of breaking here?

Reply:
Yea that makes sense